pkg/domain/domain.go

```diff
@@ -1861,6 +1861,22 @@ func (do *Domain) GetPDHTTPClient() pdhttp.Client {
 	return nil
 }
 
+func decodePrivilegeEvent(resp clientv3.WatchResponse) []string {
+	var userList []string
+	if len(resp.Events) > 0 {
+		for _, event := range resp.Events {
+			if event.Kv != nil {
+				val := event.Kv.Value
+				if len(val) > 0 {
+					users := strings.Split(string(val), ",")
```

Comment:
what if username contains `,`? And also it's hard to know the format of data rest at etcd. Maybe add comments or create a separate type for it